cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -35,13 +35,17 @@ import (
 	"go.uber.org/automaxprocs/maxprocs"
 	"golang.org/x/net/trace"
 
+	sglog "github.com/sourcegraph/log"
+
 	"github.com/google/zoekt"
 	"github.com/google/zoekt/build"
 	"github.com/google/zoekt/debugserver"
 	"github.com/google/zoekt/internal/profiler"
 )
 
 var (
+	logger sglog.Logger
```

Comment:
There should never be a package-scoped logger: https://docs.sourcegraph.com/dev/how-to/add_logging#core-concepts - loggers should be passed around explicitly (https://github.com/sourcegraph/zoekt/pull/374/files/538f85a77c8b81fe2e4fdea686571a9a4007589b#r910408258)

Reply:
Removed the package scoped logger